src/main/java/org/spongepowered/api/event/cause/EventContextKeys.java

```diff
@@ -45,7 +45,7 @@
  */
 public final class EventContextKeys {
 
-    public static final EventContextKey<UUID> CREATOR = createFor("CREATOR");
+    public static final EventContextKey<User> CREATOR = createFor("CREATOR");
```

Comment:
I suspect this was returning the UUID so that we didn't necessarily have to do a ton of lookups/disk reads to create a User unless plugins specifically asked for it.